File: source/code/php/account_management/create_account_in_dir_req.php
$account = \Stormpath\Resource\Account::instantiate([
    'username' => 'jlpicard',
    'givenName' => 'Jean-Luc',
    'surname' => 'Picard',
    'email' => 'capt@enterprise.com',
    'password' => 'uGhd%a8Kl!'
]);

$account = $directory->createAccount($account);

File: source/code/vbnet/account_management/create_account_in_dir_req.vb
Dim picard = client.Instantiate(Of IAccount)() _
    .SetUsername("jlpicard") _
    .SetEmail("capt@enterprise.com") _
    .SetGivenName("Jean-Luc") _
    .SetSurname("Picard") _
    .SetPassword("uGhd%a8Kl!")
Await captainsDirectory.CreateAccountAsync(picard)
